portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -794,6 +794,13 @@ definition {
 			value1 = "${fieldValue}");
 	}
 
+	macro viewFieldLabelByDataFieldName {
```

Comment:
@fmarcio, do you really need to create this macro or the macro `viewFieldLabel` can help you here? Can you please double check?

Reply:
> @fmarcio, do you really need to create this macro or the macro `viewFieldLabel` can help you here? Can you please double check?

@natocesarrego Unfortunately, `viewFieldLabel` did not work on my test. The test adds field labels with the same name, and the macro you suggested does not filter according to position :-(

I tried some labels macros who uses index position but none of them worked either. So I had to create this one which filters according to the data-field-name :-)